finished iteration2!

LlamaBack opened the pull request:
Co-authored_by: Erik <riggs.erik@gmail.com>
Co-authored_by: Taryn <taryn.orlemann@gmail.com>

LlamaBack committed:
finished iteration2git add *!

Co-authored_by: Erik <riggs.erik@gmail.com>
Co-authored_by: Taryn <taryn.orlemann@gmail.com>

eriggs0207 reviewed (approved):
Nice work using the team_info method to convert the team_id to the the team_name string.  Moving on to iteration 3 refactoring we can use the team_info hash to convert team info instead of reading over multilpe CSV files within the same method.  Possibly creating a team info module.

mballantyne3 commented on lib/stat_tracker.rb:
```diff
@@ -544,4 +544,82 @@ def most_goals_scored(team_id)
+  end
+
+  def rival(team_id)
+    opponent_stats = Hash.new { |opponent_id, stats| opponent_id[stats] = [0.0, 0.0, 0.0] } #[games_played, games_won]
```
I noticed that the array takes in 3 elements, and agree this could possibly come into play when trying to refactor and add different classes.

LlamaBack committed:
added co-author lines in dtr

Co-authored_by: Erik <riggs.erik@gmail.com>
Co-authored_by: Taryn <taryn.orlemann@gmail.com>

Co-authored_by: Jake <imjakekim@gmail.com>
Co-authored_by: Mary <mballantyne777@gmail.com>

mballantyne3 commented on lib/stat_tracker.rb:
```diff
@@ -544,4 +544,82 @@ def most_goals_scored(team_id)
 
+
+  def fewest_goals_scored(team_id)
+    lowest_goal = 100000
```
I wonder if there is a cleaner way to implement lowest_goal. I agree 100000 is entirely unattainable, I'm wondering if there is maybe a more readable way to do this? I'll look it up!

torlemann commented on spec/stat_tracker_spec.rb:
```diff
@@ -382,4 +382,93 @@
+
+    fake_team_info = [
+      { :teamname => "Houston Dynamo",
+        :team_id => "3",
```
I'm still a little confused as to why we can't include the ids in the fake_games hash
